app/javascript/components/molecules/DropdownMenu.js

```diff
@@ -18,7 +18,7 @@ class DropdownMenu extends React.Component {
 
     return (
       <ul
-        className={`dropdown-menu ${classes.join(' ')}`}
+        className={`dropdown-menu ${classes ? classes.join(' ') : ''}`}
```

Comment:
Same thing here it could be improved with https://github.com/JedWatson/classnames like